doc/theory/glossary.tex

```diff
@@ -65,7 +62,7 @@
 	
 	A set $\{v_0, \dots, v_n\} \subset \mathbb{R}^N$ is said to be \textit{geometrically independent} if the vectors $\{v_0-v_1, \dots, v_0-v_n\}$ are linearly independent. In this case, we refer to their convex closure as a \textit{simplex}, explicitly
 	\begin{equation*}
-	\lbrack v_0, \ldots , v_n \rbrack = \left\{ \sum c_i (v_0 - v_i)\ \big|\ c_1+\dots+c_n = 1,\ c_i \geq 0 \right\}
+	[v_0, \dots , v_n] = \left\{ \sum c_i (v_0 - v_i)\ \big|\ c_1+\dots+c_n = 1,\ c_i \geq 0 \right\}
```

Comment:
\lbrac and \rbrac are necessary unfortunately...

Reply:
Oh, ok. Everywhere?

Reply:
A small correction.
No need to do it everywhere: I introduced the change only here, because it was not properly parsed by pandoc: it seems like it has a problem with `[` at the beginning of the equation. For example, `d_i[...]...` is ok.